draft-ietf-quic-qpack.md

```diff
@@ -923,10 +961,9 @@ header field name matches the header field name of an entry stored in the static
 table or the dynamic table.
 
 If the entry is in the static table, or in the dynamic table with an absolute
-index less than or equal to Base Index, this representation starts with the '01'
-two-bit pattern.  If the entry is in the dynamic table with an absolute index
-greater than Base Index, the representation starts with the '0000' four-bit
-pattern.
+index less than the Base, this representation starts with the '01' two-bit
+pattern.  If the entry is in the dynamic table with an absolute index greater
+than the Base, the representation starts with the '0000' four-bit pattern.
```

Comment:
Something is off here because it doesn't say what to do if Base and the Absolute Index of the entry are equal.  If Base is the Insert Count when encoding started, then we need to use post-base indexing if the entry is greater than or equal to base.